### PR TITLE
Jeremy/eng 989 - Listing Form updates

### PR DESCRIPTION
**Pull request by jerminatorhits**
## Description
Why did you write this code?
update listing help section
- include 1 good and 1 bad photo instead of 2 good ones.
- more listing title examples
- change Button into Audio Loading so navigating between forms is more fluid

- Lmk if there are any suggestion on more things to add/change!

## How to Test
- [ ] Visit host/listings/:id/edit OR https://dev03.beenest.io/host/listings/256/listing_info
- [ ] See help section for `listing title` and `cover photo`
- [ ] Ensure it displays well with various screen sizes.

Which devices did you test on?
- [x] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?
Add dynamic disclaimer at bottom to let ppl know if the listing is ready to publish. This is done on the backend to make sure fields exist and is attached to the `canPublish` attribute. However, the front end would need a real-time evaluator based on the form field values to execute this feature well. This function will need to match the backend criteria and add another second place to maintain what publish validity means. Will decide if this is worth doing and possibly submit it in a future PR

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/components/routes/Host/ListingForm/ListingHelp.tsx [hunk 1]
```diff
@@ -23,7 +23,10 @@ const HomeType = () => (
 const Title = () => (
   <div>
     <h3>Listing Name</h3>
-    <p>&quot;Beautiful Home with Wifi and Views&quot;</p>
+    <h3>Choose a name that perfectly describes your listing. Be sure to use proper casing.</h3>
+    <p>&quot;Beautiful Home with View of Manhattan Skyline&quot;</p>
+    <p>&quot;Convenient Room in the Heart of San Francisco&quot;</p>
+    <p>&quot;Downtown LA Apartment near Pershing Square&quot;</p>
```
Comment (woeltjen) [inline src/components/routes/Host/ListingForm/ListingHelp.tsx]: The fontography here gets a little funny with the two `h3`s

What about `<h3>` for Listing Name, `<p>` for "Choose a name...", then maybe a `<ul>` for the examples? That seems consistent with the other help text

**Comment by jerminatorhits** [comment]
lol sry. will make those changes quick!

File: src/components/routes/Host/ListingForm/ListingHelp.tsx (hunk 1 above)
Comment (jerminatorhits) [reply src/components/routes/Host/ListingForm/ListingHelp.tsx]: good idea. thanks!
